File: Modul4/Soal 4/PRAK404-2310817210020-MuhammadHasbiyanRusyadi.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    P = int(input("Masukkan Pilihan: "))
    if 1 <= P <= 4:
        n1 = float(input("Masukkan nilai pertama: "))
        n2 = float(input("Masukkan nilai kedua: "))
        if P == 1:
            print(f"Hasil Penjumlahan antara {n1:.2f} dengan {n2:.2f} adalah {n1 + n2:.2f}")
        elif P == 2:
            print(f"Hasil Pengurangan antara {n1:.2f} dengan {n2:.2f} adalah {n1 - n2:.2f}")
        elif P == 3:
            print(f"Hasil Perkalian antara {n1:.2f} dengan {n2:.2f} adalah {n1 * n2:.2f}")
        elif P == 4:
            if n2 != 0:
                print(f"Hasil Pembagian antara {n1:.2f} dengan {n2:.2f} adalah {n1 / n2:.2f}")
            else:
                print("Error: Pembagian dengan nol tidak diizinkan.")
    elif P == 5:
        print("Terimakasih, telah menggunakan kalkulator Muhammad Hasbiyan Rusyadi")
        break
    else:
        print("Input anda salah, silahkan coba lagi")

File: Modul4/Soal 4/PRAK404-2310817210020-MuhammadHasbiyanRusyadi.c
#include<stdio.h>
int main() {
    int P;
    float n1,n2;
    do {printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &P);
        if (P >= 1 && P <= 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &n1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &n2);
            switch (P) {
                case 1:
                    printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", n1, n2, n1 + n2);
                    break;
                case 2:
                    printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", n1, n2, n1 - n2);
                    break;
                case 3:
                    printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", n1, n2, n1 * n2);
                    break;
                case 4:
                    if (n2 != 0) {
                        printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", n1, n2, n1 / n2);} 
                        else {
                        printf("Error: Pembagian dengan nol tidak diizinkan.\n");}
                    break;}} 
        else if (P == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Muhammad Hasbiyan Rusyadi\n");} 
        else {
            printf("Input anda salah, silahkan coba lagi\n");}} 
    while (P != 5);}